client/interfaces/post.ts

```diff
@@ -1,6 +1,6 @@
-import type Author from './author'
+import type Author from './author';
 
-type PostType = {
+interface PostType {
```

Comment:
Could you just make sure if the application is still running?

Reply:
ok